configuration/pih/petl/jobs/datawarnings/aggregrate/job.yml

```diff
@@ -9,7 +9,7 @@ configuration:
     table: "hivmigration_aggregrate_data_warnings"
     schema: "datawarnings/aggregrate/target.sql"
 
-  dropAndRecreateTable: false
+  dropAndRecreateTable: true
```

Comment:
We need to drop and recreate the table because of the schema change... will switch this back to false after the first build.